jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  verbose: true,
+};
```

Comment:
it would be nice if you set up you ide to add new lines at the end. However it's not crucial, it's more project code style